SmartOrder.csproj

```diff
@@ -46,59 +41,94 @@
     <PackageReference Include="SmartOrderApi" Version="1.*-*" />
   </ItemGroup>
 
+  <!--
+    Use local node_modules binaries if they exist.
+
+    If there is another directory, say in the CI/CD environment, that tailwindcss
+    is stored, then we should add another <Exec> to this block with the condition
+    that it only run when `'$(CI)' == 'true'`.  That will ensure that this can find
+    the bin anytime
+   -->
+  <Target Name="UseLocalNodeModules" Condition="Exists('node_modules/.bin')">
+    <Exec Command="export PATH=&quot;$(MSBuildProjectDirectory)/node_modules/.bin:$PATH&quot;"  />
+  </Target>

```

Comment:
If we're using tailwind, we're using nodejs.  Not really a great way around that unless you want to bundle it up into its own binary, which means it would have to be rebundled for other operating systems and architectures.

Reply:
And yeah, that's for the cases where it gets installed locally.  Standard procedure would be to have it installed locally in the project, so that the project is more portable, but with the container, it wont' be an issue anyway.  It will be doing it's `npm install` dealio automatically, and everything will be really nice and clean.  If we do ever use any other nodejs utilties, then node modules will be needed